app/controllers/offers_controller.rb

```diff
@@ -0,0 +1,26 @@
+class OffersController < ApplicationController
+  def index
+    @offers = Offer.all
+  end
+
+  def new
+    @offer = Offer.new(offer_params)
+  end
+
+  def create
+    @listing = Listing.find(params[:listing_id])
+    @offer = @listing.offers.create(offer_params)
```

Comment:
do you need to use an instance variable here?

Reply:
didnt work without

Reply:
Please explain what is the other option you tried @Carrosen